data/tutorials/getting-started/1_01_a_tour_of_ocaml.md

```diff
@@ -11,13 +11,26 @@ recommended_next_tutorials:
   - "lists"
 ---
 
-Before proceeding with this tutorial, please ensure you've installed OCaml and set up the environment, as described on the [install OCaml](/docs/installing-ocaml) page. After we take an introductory tour of OCaml's language features, we'll proceed to create our first OCaml project in the [Your First OCaml Program](/docs/your-first-program) tutorial.
+## Introduction
+Hop on the OCaml sightseeing bus.
+This absolute beginner tutorial will drive you through the marvels and wonders of OCaml.
+We'll have a look at the most commonly used language features.
```

Comment:
```suggestion
In this tutorial, we'll have a look at the most commonly used language features.
```
Let's make this more boring.

Reply:
I liked it fun. My initial thought was the bits about "hop on..." should not be just at the summary, but also in the text itself. But ok, let's make it a bit more boring. 
I think It might be a good idea to work more on the introduction after we go thru the text (in another PR)